explanation/security/charms.md

```diff
@@ -27,3 +27,33 @@ When Anbox Cloud is deployed without the use of an external CA, the charms will
 ### Packages used
 
 * [cryptography from PyPI](https://pypi.org/project/cryptography/)
+
+## Juju secrets
+
```

Comment:
Don't we have to provide additional steps in the upgrade guide?

Reply:
Since Juju secret support is intended to be transparent to the end user,  why do we have to provide additional steps in the upgrade guide? or specifically, you mean for the cross model case?